### PR TITLE
generate: rectify the arguments

### PR DESCRIPTION
**Pull request by q384566678**
Since in https://github.com/opencontainers/runtime-tools/issues/343#issuecomment-286005046 @Kxuan  put forward the idea, I submitted this PR.

Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/oci-runtime-tool/generate.go [hunk 1]
```diff
@@ -735,23 +735,23 @@ func parseDevice(device string, g *generate.Generator) (rspec.LinuxDevice, error
 func addSeccomp(context *cli.Context, g *generate.Generator) error {
 
 	// Set the DefaultAction of seccomp
-	if context.IsSet("seccomp-default") {
-		seccompDefault := context.String("seccomp-default")
+	if context.IsSet("linux-eccomp-default") {
```
Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: linux-eccomp  -> linux-seccomp

File: cmd/oci-runtime-tool/generate.go [hunk 2]
```diff
@@ -21,77 +21,77 @@ var generateFlags = []cli.Flag{
...
+	cli.StringSliceFlag{Name: "process-cap-add", Usage: "add Linux capabilities"},
+	cli.StringSliceFlag{Name: "process-cap-drop", Usage: "drop Linux capabilities"},
+	cli.StringFlag{Name: "process-cwd", Value: "/", Usage: "current working directory for the process"},
+	cli.IntFlag{Name: "process-gid", Usage: "gid for the process"},
+	cli.StringSliceFlag{Name: "process-groups", Usage: "supplementary groups for the process"},
+	cli.BoolFlag{Name: "process-no-new-privileges", Usage: "set no new privileges bit for the container process"},
+	cli.StringSliceFlag{Name: "process-rlimits-add", Usage: "specifies resource limits for processes inside the container. "},
+	cli.StringSliceFlag{Name: "process-rlimits-remove", Usage: "remove specified resource limits for processes inside the container. "},
+	cli.BoolFlag{Name: "process-rlimits-remove-all", Usage: "remove all resource limits for processes inside the container. "},
+	cli.StringFlag{Name: "process-selinux-label", Usage: "process selinux label"},
```
Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: selinuxlabe is Linux-specific, linux-selinux-label will be better

File: cmd/oci-runtime-tool/generate.go [hunk 3]
```diff
@@ -21,77 +21,77 @@ var generateFlags = []cli.Flag{
 	cli.StringFlag{Name: "apparmor", Usage: "specifies the the apparmor profile for the container"},
```
Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: apparmor is also Linux-specific

File: cmd/oci-runtime-tool/generate.go (hunk 2 above)
Comment (q384566678) [inline cmd/oci-runtime-tool/generate.go]: `selinuxLabel` is part of the `process`, I think the `process-` will be better. `linux-selinux-label`may be confusing.

Comment (Mashimiao) [inline cmd/oci-runtime-tool/generate.go]: As I said it's Linux-specific, not for all platforms.

Comment (q384566678) [inline cmd/oci-runtime-tool/generate.go]: updated.

**Comment by Mashimiao** [comment]
need rebase

**Comment by q384566678** [comment]
rebased.

**Comment by Mashimiao** [comment]
LGTM

**Comment by Mashimiao** [comment]
need rebase

**Comment by q384566678** [comment]
updated

**Comment by Mashimiao** [comment]
@liangchenye  @mrunalp  @hqhq  PTAL

**Comment by q384566678** [comment]
@liangchenye @hqhq @mrunalp PTAL

**Comment by liangchenye** [comment]
LGTM

**Comment by Mashimiao** [comment]
LGTM

**Pull request merged**